src/util/common/config.cpp

```diff
@@ -682,6 +682,11 @@ namespace cbdc::config {
                 return "Two-phase mode requires at least one configured "
                        "sentinel";
             }
+            if(opts.m_sentinel_endpoints.size()
```

Comment:
It looks like this commit added this check but didn't remove it from `::init()`; is it valuable having this check twice?

Reply:
Nope--it's probably not valuable.  I just forgot about it.  I just refactored the commits to remove it and to hopefully improve the commit messages.